pkg/operator/const.go

```diff
@@ -9,4 +9,8 @@ const (
 
 	Namespace  = "openshift-azure-operator"
 	SecretName = "cluster"
+
+	OperatorIdentityName       = "ServiceOperator"
+	OperatorIdentitySecretName = "azure-cloud-credentials"
+	OperatorTokenFile          = "/var/run/secrets/openshift/serviceaccount/token"
```

Comment:
I just happened to have reviewed https://github.com/Azure/ARO-RP/pull/3776 - is there any way this const could be used in both places? It seems like these two PRs are very much related - one, generating the tokens, the other, mounting them.

Reply:
I'll rebase #3776 onto this PR and pass `OperatorTokenFile` in to the template